BangazonAPI/Models/Customer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class Customer
+    {
+        public int id { get; set; }
+        public string firstName { get; set; }
+        public string lastName { get; set; }
+
+        public DateTime AccountCreated { get; }
+        public DateTime lastActive { get; }
+
+        public List<PaymentType> PaymentType { get; set; } = new List<PaymentType>
```

Comment:
list should have plural names-- let's name this one `PaymentTypes` instead of `PaymentType`